config/k9s.yml

```
@@ -0,0 +1,76 @@
+    # enable TTY
+    tty: true
+    # the shell pod image to use
+    image: killerAdmin
+    # the namespace to launch to shell pod into
+    namespace: default
+    # the resource limit to set on the shell pod
+    limits:
+      cpu: 100m
+      memory: 100Mi
+    hostPathVolume:
+    - name: docker-socket
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)